core/src/test/com/es/core/dao/impl/JdbcPhoneDaoTest.java

```diff
@@ -0,0 +1,7 @@
+package com.es.core.dao.impl;
+
+import static org.junit.Assert.*;
+
+public class JdbcPhoneDaoTest {
```

Comment:
Interesting way of fixing tests... Hope you will not use such approach in real project:)